### PR TITLE
updata PPO/MADDPG parl version

### PR DESCRIPTION
**Pull request by ShuaibinLi**

### CODE REVIEW

File: benchmark/torch/maddpg/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ simple_world_comm<br>
 ### Dependencies:
 + python3.5+
 + torch
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl==2.0.6](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline benchmark/torch/maddpg/README.md]: `>=`

File: benchmark/torch/ppo/README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Please see [mujoco-py](https://github.com/openai/mujoco-py) to know more about M
 ## How to use
 ### Dependencies:
 + python>=3.6.2
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl=2.0.6](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline benchmark/torch/ppo/README.md]: `==`

File: examples/MADDPG/README.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ simple_world_comm<br>
 ### Dependencies:
 + python3.5+
 + [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl==2.0.6](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline examples/MADDPG/README.md]: `>=`

File: examples/MADDPG/requirements.txt [hunk 4]
```diff
@@ -1,4 +1,4 @@
 PettingZoo==1.17.0
 gym==0.23.1
-parl
-paddlepaddle
+parl==2.0.6
```
Comment (rical730) [inline examples/MADDPG/requirements.txt]: `>=`

File: benchmark/torch/maddpg/README.md (hunk 1 above)
Comment (ShuaibinLi) [reply benchmark/torch/maddpg/README.md]: done

File: benchmark/torch/ppo/README.md (hunk 2 above)
Comment (ShuaibinLi) [reply benchmark/torch/ppo/README.md]: done

File: examples/MADDPG/README.md (hunk 3 above)
Comment (ShuaibinLi) [reply examples/MADDPG/README.md]: done

File: examples/MADDPG/requirements.txt (hunk 4 above)
Comment (ShuaibinLi) [reply examples/MADDPG/requirements.txt]: done

File: benchmark/torch/ppo/README.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ Please see [mujoco-py](https://github.com/openai/mujoco-py) to know more about M
 ## How to use
 ### Dependencies:
 + python>=3.6.2
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl==2.0.6](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline benchmark/torch/ppo/README.md]: parl>=2.0.6

File: examples/PPO/README.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ Please see [mujoco-py](https://github.com/openai/mujoco-py) to know more about M
 ## How to use
 ### Dependencies:
 + [paddle>=2.3.1](https://github.com/PaddlePaddle/Paddle)
-+ [parl>2.0.5](https://github.com/PaddlePaddle/PARL)
++ [parl==2.0.6](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline examples/PPO/README.md]: parl>=2.0.6

File: examples/PPO/requirements.txt [hunk 7]
```diff
@@ -1,4 +1,4 @@
 gym==0.21.0
 mujoco-py==2.1.2.14
-paddlepaddle>=2.0.0
-parl>2.0.5
+paddlepaddle>=2.3.1
+parl==2.0.6
```
Comment (rical730) [inline examples/PPO/requirements.txt]: parl>=2.0.6

File: benchmark/torch/ppo/README.md (hunk 5 above)
Comment (ShuaibinLi) [reply benchmark/torch/ppo/README.md]: done

File: examples/PPO/README.md (hunk 6 above)
Comment (ShuaibinLi) [reply examples/PPO/README.md]: done

File: examples/PPO/requirements.txt (hunk 7 above)
Comment (ShuaibinLi) [reply examples/PPO/requirements.txt]: done

**Pull request closed without merging**